plugins/translate.js

```diff
@@ -1231,25 +1231,26 @@ translate = {
                       columns:[ { name:'tag1',
                                   desc:'Tag List',
                                   type:'String',
+                                  length:'8192'
```

Comment:
Just asking for a little context, why is this one `8192` instead of the `254` commented out below while all the rest of the `tag*` fields are limited to `254`?

Reply:
We really want to put all of the left over tags into one attribute and 8K should be big enough.

Tag[2-4] are for when writing to shapefile which has a 256 char max for each attribute. 

There is yet-another-command-line-switch that says to split the o2s values into tag[1-4]
```
=== ogr.split.o2s

* Data Type: bool
* Default Value: `false`

If the list of o2s tags is > 255 char, split it into into 254 char long pieces. If this is false,
it will be exported as one big string.
```

Originally, we didn't specify a length for these attributes and just used the default one from the GDAL driver. With the addition of length checking, we needed to assign a value to avoid the shapefile default of 80 characters for the field width..

Reply:
I see, that makes sense.